.github/actions/package-windows-qt/action.yml

```diff
@@ -66,14 +66,16 @@ runs:
     - name: Create combined Qt archive
       shell: pwsh
       run: |
+        if ( '${{ github.event_name }}' -eq 'push' -and '${{ github.ref }}' -like '*refs/tags/*' ) { $CompressionLevel = '9' } else { $CompressionLevel = '5' }
```

Comment:
This is fine, you can also check `github.ref_name` for the actual tag provided.

Reply:
I think I just copied this from our main.yml.